.github/workflows/snyk-pr.yml

```diff
@@ -0,0 +1,15 @@
+name: Snyk PR
+on:
+    pull_request:
+        paths:
+            - 'project/Dependencies.scala'
+            - 'project/plugins.sbt'
+            - 'package.json'
```

Comment:
Is it worth including these 2 files in this list:
https://github.com/guardian/frontend/blob/main/project/ProjectSettings.scala
https://github.com/guardian/frontend/blob/main/build.sbt

ProjectSettings specifically, as it touches the scala version which if changed would impact what dependencies SBT pulls in

Reply:
Yea, `ProjectSettings` makes sense, `build.sbt` has no deps in it, so would be wary of adding it. It might be nice to see if we can get this repo into a more [standard sbt project structure](https://www.scala-sbt.org/1.x/docs/Multi-Project.html) post-upgrade.